packages/compass-e2e-tests/helpers/connect-form-state.ts

```diff
@@ -0,0 +1,84 @@
+// undefined if not set/existing, boolean for ticked checkboxes, strings for
+// everything else because this is the raw form field state as much as possible
+export interface ConnectFormState {
+  // Default
+  connectionString?: string;
+  scheme?: 'MONGODB' | 'MONGODB_SRV';
+  hosts?: string[];
+  directConnection?: boolean;
+
+  // Authentication
+  authMethod?:
+    | 'AUTH_NONE'
```

Comment:
what's `AUTH_NONE`?

Reply:
That's what the form calls that radio button. These all map to the form state.

Reply:
I see, makes sense